app/views/shared/_navbar.html.erb

```diff
@@ -23,12 +23,12 @@
           </a>
           <ul class="dropdown-menu dropdown-menu-right menu-options pb-0">
             <li>
-              <%= link_to forum_link, class: "drop-down-link accent" do %>
+              <%= link_to forum_link, class: 'drop-down-link accent', target: '_blank' do %>
```

Comment:
Can we add `rel: 'noopener noreferrer'` as well?

https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
@mindovermiles262 Done :)